To-Do-app/src/App.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+import './App.css'
+import Header from './Components/Header'
+import AddTask from './Components/AddTask'
+import PowerSettingsNewIcon from '@mui/icons-material/PowerSettingsNew'
+import Dialog from '@mui/material/Dialog'
+import DialogActions from '@mui/material/DialogActions'
+import DialogContent from '@mui/material/DialogContent'
+import DialogContentText from '@mui/material/DialogContentText'
+import DialogTitle from '@mui/material/DialogTitle'
+import { Button, makeStyles } from '@material-ui/core'
+import { useDispatch } from 'react-redux'
+import { login, logout } from './features/userSlice'
+import { useSelector } from 'react-redux'
+import { selectUser } from './features/userSlice'
+
+const useStyles = makeStyles({
+  loginText: {
+    textAlign: 'center',
+    fontSize: '40px',
+    color: 'white',
+    position: 'relative',
+    top: '210px'
+  }
+})
+
+// const successLocalStorage = JSON.parse(localStorage.getItem("success") || "[]")
+
+function App() {
+
+  const classes = useStyles()
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const [open, setOpen] = useState(false)
+
+  const { loggedIn } = useSelector(selectUser)
+  const { username } = useSelector(selectUser)
```

Comment:
why you need 2 useSelector here ? cant you achieve the same in one statement ?

Reply:
Yes sir, It can be done in one statement